security/pkg/pki/ca/ca.go

```diff
@@ -275,15 +275,21 @@ type IstioCA struct {
 // NewIstioCA returns a new IstioCA instance.
 func NewIstioCA(opts *IstioCAOptions) (*IstioCA, error) {
 	ca := &IstioCA{
-		defaultCertTTL: opts.DefaultCertTTL,
-		maxCertTTL:     opts.MaxCertTTL,
-		keyCertBundle:  opts.KeyCertBundle,
-		livenessProbe:  probe.NewProbe(),
+		maxCertTTL:    opts.MaxCertTTL,
+		keyCertBundle: opts.KeyCertBundle,
+		livenessProbe: probe.NewProbe(),
 	}
 
 	if opts.CAType == selfSignedCA && opts.RotatorConfig.CheckInterval > time.Duration(0) {
 		ca.rootCertRotator = NewSelfSignedCARootCertRotator(opts.RotatorConfig, ca)
 	}
+
```

Comment:
Can you add a comment explaining 1) the root cause of the problem and 2) how this PR fixes the problem?